2020-09-13 16:41:19,860 DEBUG   MainThread:16101 [wandb_config.py:_load_defaults():154] no defaults not found in config-defaults.yaml
2020-09-13 16:41:20,041 DEBUG   MainThread:16101 [meta.py:_setup_code_git():49] probe for git information
2020-09-13 16:41:20,088 DEBUG   MainThread:16101 [run_manager.py:__init__():547] Initialized sync for FDA_integration_to_INTRA_DA/1vuvcsfg
2020-09-13 16:41:20,095 INFO    MainThread:16101 [run_manager.py:wrap_existing_process():1149] wrapping existing process 16086
2020-09-13 16:41:20,096 WARNING MainThread:16101 [io_wrap.py:register():108] SIGWINCH handler was not None: <Handlers.SIG_DFL: 0>
2020-09-13 16:41:20,225 INFO    MainThread:16101 [run_manager.py:init_run():928] system metrics and metadata threads started
2020-09-13 16:41:20,225 INFO    MainThread:16101 [run_manager.py:init_run():967] upserting run before process can begin, waiting at most 10 seconds
2020-09-13 16:41:20,462 INFO    Thread-15 :16101 [run_manager.py:_upsert_run():1056] saving pip packages
2020-09-13 16:41:20,463 INFO    Thread-15 :16101 [run_manager.py:_upsert_run():1058] initializing streaming files api
2020-09-13 16:41:20,464 INFO    Thread-15 :16101 [run_manager.py:_upsert_run():1065] unblocking file change observer, beginning sync with W&B servers
2020-09-13 16:41:20,465 INFO    MainThread:16101 [run_manager.py:wrap_existing_process():1166] informing user process we are ready to proceed
2020-09-13 16:41:20,471 INFO    MainThread:16101 [run_manager.py:_sync_etc():1298] entering loop for messages from user process
2020-09-13 16:41:21,026 INFO    Thread-4  :16101 [run_manager.py:_on_file_modified():692] file/dir modified: /media/data/hlim/IntraDA/intrada/wandb/run-20200913_164119-1vuvcsfg/config.yaml
2020-09-13 16:41:21,206 INFO    Thread-4  :16101 [run_manager.py:_on_file_created():681] file/dir created: /media/data/hlim/IntraDA/intrada/wandb/run-20200913_164119-1vuvcsfg/requirements.txt
2020-09-13 16:41:21,206 INFO    Thread-4  :16101 [run_manager.py:_on_file_created():681] file/dir created: /media/data/hlim/IntraDA/intrada/wandb/run-20200913_164119-1vuvcsfg/output.log
2020-09-13 16:41:21,207 INFO    Thread-4  :16101 [run_manager.py:_on_file_created():681] file/dir created: /media/data/hlim/IntraDA/intrada/wandb/run-20200913_164119-1vuvcsfg/wandb-metadata.json
2020-09-13 16:41:23,026 INFO    Thread-4  :16101 [run_manager.py:_on_file_modified():692] file/dir modified: /media/data/hlim/IntraDA/intrada/wandb/run-20200913_164119-1vuvcsfg/output.log
2020-09-13 16:41:23,721 INFO    MainThread:16101 [run_manager.py:_sync_etc():1355] process received interrupt signal, shutting down
2020-09-13 16:41:23,722 INFO    MainThread:16101 [run_manager.py:_sync_etc():1408] closing log streams and sending exitcode to W&B
2020-09-13 16:41:23,727 INFO    MainThread:16101 [run_manager.py:shutdown():1072] shutting down system stats and metadata service
2020-09-13 16:41:24,027 INFO    Thread-4  :16101 [run_manager.py:_on_file_created():681] file/dir created: /media/data/hlim/IntraDA/intrada/wandb/run-20200913_164119-1vuvcsfg/wandb-events.jsonl
2020-09-13 16:41:24,228 INFO    MainThread:16101 [run_manager.py:shutdown():1086] stopping streaming files and file change observer
2020-09-13 16:41:25,028 INFO    Thread-4  :16101 [run_manager.py:_on_file_modified():692] file/dir modified: /media/data/hlim/IntraDA/intrada/wandb/run-20200913_164119-1vuvcsfg/wandb-metadata.json
2020-09-13 16:41:26,341 INFO    MainThread:16101 [run_manager.py:_sync_etc():1420] process only ran for 6 seconds, not syncing files
Thread-4  :16101 [1vuvcsfg:run_manager.py:_on_file_modified():692] file/dir modified: /media/data/hlim/IntraDA/intrada/wandb/run-20200913_164119-1vuvcsfg/wandb-metadata.json
2020-09-13 16:41:26,341 INFO    MainThread:16101 [1vuvcsfg:run_manager.py:_sync_etc():1420] process only ran for 6 seconds, not syncing files
